pkgs/applications/science/biology/slicer3d/default.nix

```diff
@@ -0,0 +1,119 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  makeDesktopItem,
+  copyDesktopItems,
+  glib,
+  fetchzip,
+  pulseaudio,
+  xorg,
+  zlib,
+  fontconfig,
+  libxkbcommon,
+  libGL,
+  libGLU,
+  freetype,
+  cups,
+  unixODBC,
+  libxcrypt-legacy,
+  hwloc,
+  alsa-lib,
+  postgresql,
+  nss,
+  nspr,
+}:
+
+let
+  pname = "Slicer3D";
+  version = "5.6.2";
+  stability = "release";
+  os = "linux";
+  old_hwloc = hwloc.overrideAttrs (_: {
+    version = "1.4.1";
+  });
+
+  desktopItem = makeDesktopItem {
+    name = "Slicer3D";
+    exec = "SlicerApp-real";
+    desktopName = "Slicer 3D";
+    genericName = "A 3D Slicer Program"; # edit later
+    categories = [ "Graphics" ]; # Edit later
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version;
+  src =
+    let
+      name = "Slicer-${version}-linux-amd64.tar.gz";
+      url = "http://download.slicer.org/download?os=${os}&stability=${stability}&version=${version}";
+    in
+    fetchzip {
+      inherit name;
+      inherit url;
+      hash = "sha256-DmJS1yrwJBcAIRnVA8VdsO4u82E1MWX+uvRC+6dEXmM=";
+      extension = "tar.gz";
+    };
+
+  desktopItems = [
+    desktopItem
+  ];
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    copyDesktopItems
+  ];
+
+  buildInputs = [
+    freetype
+    xorg.libSM
+    xorg.libICE
+    xorg.libXrender
+    xorg.libXext
+    xorg.libX11
+    zlib
+    stdenv.cc.cc.lib
+    libGL
+    libGLU
+    fontconfig
+    xorg.xkbutils
+    xorg.xcbutil
+    xorg.xcbutilwm
+    xorg.xcbutilimage
+    xorg.xcbutilrenderutil
+    xorg.xcbutilkeysyms
+    xorg.libXdamage
+    xorg.libXfixes
+    xorg.libXrandr
+    xorg.libXcursor
+    xorg.libXtst
+    xorg.libXcomposite
+    alsa-lib
+    postgresql
+    libxkbcommon
+    glib
+    pulseaudio
+    cups
+    unixODBC
+    libxcrypt-legacy
+    old_hwloc
+    nss
+    nspr
+  ];
```

Comment:
nit: although not required by `nixfmt`, consider adding some additional (single) newlines between attributes.